stable/appmesh-controller/README.md

```diff
@@ -267,4 +267,5 @@ Parameter | Description | Default
 `enableCertManager` |  Enable Cert-Manager | `false`
 `xray.image.repository` | X-Ray image repository | `amazon/aws-xray-daemon`
 `xray.image.tag` | X-Ray image tag | `latest`
+`accountId` | AWS Account ID must be provided for a VPC without internet access | None
```

Comment:
The current description may not be accurate in all cases. Can we please change it to something like: `AWS Account ID for the Kubernetes cluster`

Reply:
Updated the description as suggested, however I'll recommend to specify parameter is needed for Private VPCs (maybe in the release notes?)

Reply:
Thanks! +1